__tests__/apiArticles.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/articles", () => {
+  it("responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toBeInstanceOf(Object);
+
+        const expectedProperties = [
+          "article_id",
+          "title",
+          "topic",
+          "created_at",
+          "votes",
+          "article_img_url",
+          "comment_count",
+        ];
+        const articles = response.body.articles
+
+        response.body.articles.forEach((article) => {
```

Comment:
This is ok but toMatchObject would be cleaner and easier for you. Have a look at it when you get the chance.